runtimes/aleph-debian-11-python/init1.py

```diff
@@ -62,9 +73,9 @@ class ConfigurationPayload:
     input_data: bytes
     interface: Interface
     vm_hash: str
-    code: bytes = None
-    encoding: Encoding = None
-    entrypoint: str = None
+    code: bytes
+    encoding: Encoding
+    entrypoint: str
```

Comment:
If these are always specified, okay. I don't see what these lines have to do with this PR though.

Reply:
Moved to https://github.com/aleph-im/aleph-vm/pull/408